documentation/book/proc-renewing-your-own-ca-certificates.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='renewing-your-own-ca-certificates-{context}']
+= Renewing your own CA certificates
+
+This procedure will show you how to renew CA certificates and private keys you installed previously.
```

Comment:
_Please reword as follows:_

This procedure describes how to renew CA certificates and private keys that you previously installed.